tests/baselines/reference/classStaticBlock18(target=es2015).js

```diff
@@ -0,0 +1,33 @@
+//// [classStaticBlock18.ts]
+function foo () {
+  return class {
+    static foo = 1;
+    static {
+      const c = class {
+        static bar = 2;
+        static {
+          // do
+        }
+      }
+    }
+  }
+}
+
+
+//// [classStaticBlock18.js]
+function foo() {
+    var _a, _b, _c__, __;

```

Comment:
The `_b` and `_c__` variables should be scoped to the static block IIFE for the outer class expression. You may be missing a call to `startLexicalEnvironment`/`endLexicalEnvironment` in the transform somewhere.